src/ui/public/autocomplete_providers/index.js

```diff
@@ -17,22 +17,12 @@
  * under the License.
  */
 
-export function escapeQuotes(string) {
-  return string.replace(/"/g, '\\"');
-}
-
-export const escapeKuery = (string) => escapeNot(escapeAndOr(escapeSpecialCharacters(string)));
-
-// See the SpecialCharacter rule in kuery.peg
-function escapeSpecialCharacters(string) {
-  return string.replace(/[\\():<>"*]/g, '\\$&'); // $& means the whole matched string
-}
+const autocompleteProviders = new Map();
```

Comment:
This is basically a registry, but I didn't actually want to use a `registry` because it relies on `Private` which would make it difficult to include in APM.